compiler/src/iree/compiler/Codegen/SPIRV/SPIRVTile.cpp

```diff
@@ -136,8 +136,8 @@ static LogicalResult tileAndDistributeToThreads(linalg::LinalgOp consumerOp,
   const char *attrName = getSPIRVDistributeAttrName();
   // We can have more than 3 dimensions being tiled (e.g., for convolutions with
   // non-1 batch). But only the innermost 3 dimensions are distributed.
-  for (int i = loops.size() - 1, dim = 0; i >= 0 && dim < 3; --i) {
-    loops[i]->setAttr(attrName, rewriter.getIndexAttr(dim++));
+  for (auto [dim, loop] : zip(llvm::seq(0, 3), llvm::reverse(loops))) {
+    loop->setAttr(attrName, rewriter.getIndexAttr(dim++));
```

Comment:
nit: not need for `dim++` now

Reply:
Oops. https://github.com/openxla/iree/pull/13047 to fix this.